File: fixtures/languages/python/simple_module.py
"""Minimal Python fixture for T1 golden conformance.

Covers: function defs, import, same-file call, unresolved call, class + extends.
"""
from os import path as ospath


def helper(x):
    return x + 1


class Base:
    pass


class Child(Base):
    def run(self):
        return helper(1)


def main():
    helper(2)
    missing_fn()

File: fixtures/languages/perl/simple_module.pl
# Minimal Perl fixture for T1 golden conformance.
#
# Covers: package, use imports, sub defs, same-file call, unresolved call.

package Sample;
use strict;
use warnings;
use File::Basename;

sub helper {
    my ($x) = @_;
    return $x + 1;
}

sub main {
    helper(2);
    missing_fn();
}

1;
